src/main/java/org/minions/devfund/david/Movie/MovieDatabase.java

```diff
@@ -0,0 +1,110 @@
+package org.minions.devfund.david.Movie;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)